.github/workflows/i18n-crowdin-download.yml

```diff
@@ -0,0 +1,53 @@
+name: Crowdin Download Action
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 * * * *" # TODO: Do we want to run this every hour?
+
+jobs:
+  download-sources-from-crowdin:
+    runs-on: ubuntu-latest
+
+# TODO: For the test repo this is not required. Maybe for grafana/grafana?
+#    permissions:
+#      contents: write # needed to commit changes back into the PR
+#      pull-requests: write # needed to update PR description
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Download sources
+        uses: crowdin/github-action@v1
+        with:
+          upload_sources: false
+          upload_translations: false
+          download_sources: false
+          download_translations: true
+          localization_branch_name: l10n_crowdin_translations
+          create_pull_request: true
+          pull_request_title: 'I18n: Download translations from Crowdin'
+          pull_request_body:  |
+            :robot: Automatic download of translations from Crowdin.
+
+            Steps for merging:
+              1. Wait for the **New Crowdin translations by GitHub Action** commit that may be required for CI to pass.
```

Comment:
This step should not be required any more as this action should run it before committing to the PR.